ppx/ppx_deriving_json/tests/main.ml

```diff
@@ -1,5 +1,9 @@
-let _ = Deriving_Json.read
+let mapper _args =
+  let structure _ st =
+    Ppxlib_ast.Selected_ast.of_ocaml Structure st
+    |> Ppxlib.Driver.map_structure
+    |> Ppxlib.Selected_ast.to_ocaml Structure
+  in
+  { Ast_mapper.default_mapper with structure }
 
-let _ = Ppx_deriving_json.json
-
-let () = Toplevel_expect_test.run Migrate_parsetree.Driver.run_as_ast_mapper
+let () = Toplevel_expect_test.run mapper
```

Comment:
note that the mapper will not behave correctly if one decide to update other fields. I think we should leave a small comment about it.

Reply:
Yes. I've done that now.